### ISSUE TITLE
Issue with Preview App

### ISSUE THREAD

**Issue by SteveMattan** [ttscoff/bunch-docs#326]
Using the Preview app in Bunch does not behave as I would expect.

The players:
macOS v 14.3.1
Bunch v 1.4.17 (180)
Preview v 11.0 (1056.3.2)

The following Bunch file fails with error dialog shown:

---

\---
title: Test Bunch
\---
\# Test Bed for Bunch Ideas

Preview
\- /Users/stevemattan/Desktop/Introducing Bunch - Bunch.pdf

---

![Screenshot 2024-03-07 at 1 08 50 PM](https://github.com/ttscoff/bunch/assets/95952224/29d28f40-5fce-4a7e-b199-26a63bb88859)

Changing **Preview** to **Preview.app** does launch Preview and opens the file. There is a file open dialog shown briefly when the Bunch is run.

However, upon closing the Bunch, Preview does not quit. Adding the line, **!!!Preview.app** does not result in Preview quitting upon close either (nor does: **!!!Preview** ).

Finally, the line:

 /Users/stevemattan/Desktop/Introducing Bunch - Bunch.app.pdf

does result in the file being opened in Preview (Preview still does not quit upon close of the Bunch). (It appears the entering the path to a file will always open the file in the default app, but I do not see that noted in the syntax reference. Sorry if I missed it.)

To recap:
- entering the app name "Preview" results in an error message.
- entering "Preview.app" successfully launches the app, and the file is opened.
- closing the Bunch does not automatically close the Preview app.
- explicitly calling for the Preview app to be quit upon closing the Bunch, with the !!! syntax, does not quit the app.
- simply entering the path to the PDF file will open the file in Preview (the default for .pdf).

Hopefully this makes sense. Please let me know if it does not, or if you need additional information.

**Comment by ttscoff** [comment]
"Preview" is being interpreted by the system as iPreview. Not really 
Bunch's fault, it's what the underlying system gives it back when it 
asks for a URL for a named app. The workaround would be to use the 
bundle id, com.apple.Preview.

**Comment by SteveMattan** [comment]
Thank you for the prompt reply. 

Alas, changing to **com.apple.Preview**, resulted in the same error dialog:

![Screenshot 2024-03-07 at 2 46 32 PM](https://github.com/ttscoff/bunch/assets/95952224/41798e8f-1445-499f-a101-6597c5817da4)

But, Preview did launch, presenting an open file dialog. Preview did not quit upon closing the Bunch.  

For my purposes I can just enter the path to the file, it is a book in PDF form, and quit Preview when I'm done.

Thanks again, and stay well.
